js/apps/admin-ui/src/dashboard/ProviderInfo.tsx

```diff
@@ -56,37 +55,39 @@ export const ProviderInfo = () => {
                 <Td>{name}</Td>
                 <Td>
                   <ul>
-                    {Object.entries(providers).map(([key]) => (
-                      <li key={key}>{key}</li>
-                    ))}
+                    {Object.entries(providers).map(
+                      ([key, { operationalInfo }]) => (
+                        <>
+                          <li key={key}>{key}</li>
+                          {operationalInfo ? (
```

Comment:
Semantically a `ul` is only allowed to have `li` as children, no other elements are allowed.